submission/programs/adsl.R

```diff
@@ -304,8 +304,8 @@ adsl07 %>%
   xportr_length(adsl_spec) %>% # Assigns SAS length from a variable level metadata
   xportr_label(adsl_spec) %>% # Assigns variable label from metacore specifications
   xportr_df_label(adsl_spec) %>% # Assigns dataset label from metacore specifications
-  xportr_format(adsl_spec$var_spec %>% 
-                  mutate_at(c("format"), ~ replace_na(., "")), "ADSL") %>%
+  xportr_format(adsl_spec$var_spec %>%
+    mutate_at(c("format"), ~ replace_na(., "")), "ADSL") %>%
```

Comment:
Thanks for updating this @kaz462 .  I see that @DeclanHodges has other ADSL issues that he is working on in #88 and #94 . 

@DeclanHodges , I'm going to go ahead and merge this branch to main which will update the `adsl.R`.  Then you can get updated script from main before working on 88 and 94.

Reply:
Apologies I haven't had time to get around to reviewing this -  thanks @kaz462 for updating this and @laxamanaj for reviewing!